portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,75 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.servlets.DataServlet;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/** Servlet that handles comments data */
+@WebServlet("/comment")
 public class DataServlet extends HttpServlet {
+    public static String NAME = "Comment";
+    public static String CONTENT = "content";
+    public static String TIMESTAMP = "timestamp";
+    public static String EMAIL = "email";
```

Comment:
We need this in a place where other classes (like DeleteDataServlet) can use it.

Reply:
A Constants class is created in sps/data folder.